docs/intents.rst

```diff
@@ -202,6 +202,6 @@ On Windows use ``py -3`` instead of ``python3``.
 
 .. warning::
 
-    There is no currently set date in which the old gateway will stop working so it is recommended to update your code instead.
+    There is no current set date on which the old gateway will stop working so it is recommended to update your code instead.
```

Comment:
This change is wrong.

Anyway it might be better to remove this section entirely since it no longer works.